package.json

```diff
@@ -45,7 +45,7 @@
         "inquirer": "8.2.2",
         "json-to-table": "4.2.1",
         "mustache": "4.2.0",
-        "p-limit": "4.0.0",
+        "p-limit": "3.1.0",
```

Comment:
why an older version?

Reply:
same logic, but version 4 uses ESM modules, which was causing issues when running our or CJS context. 
Suggestion would be to hold until a full move to ESM modules OR TS